renew-le.sh

```diff
@@ -40,7 +40,8 @@ letsencrypt certonly --standalone --csr "$WORKDIR/httpd-csr.der" --email "$EMAIL
 
 # replace the cert
 cp /var/lib/ipa/certs/httpd.crt /var/lib/ipa/certs/httpd.crt.bkp
-mv -f "$WORKDIR/0000_cert.pem" /var/lib/ipa/certs/httpd.crt
+cat "$WORKDIR/0000_cert.pem" "$WORKDIR/0000_chain.pem" > "$WORKDIR/full_chain.pem"
+mv -f "$WORKDIR/full_chain.pem" /var/lib/ipa/certs/httpd.crt
```

Comment:
Ahh well I had to do this because that chain is missing the new LE CA's. So you get cert errors when it's missing. So I guess an alternative is to add these new servers to that chain then?

Reply:
You do that in your patch. ipa-cacert-manage loads the new certs into IPA and ipa-certupdate updates the places that need the full chain, including /etc/ipa/ca.crt.

Reply:
See I just checked both of my servers and that chain is missing the new CA's. That's why I added this code, on both my servers I had SSL errors in Safari and Firefox and Qualys reported the Intermediate CA as missing from the chain.

Reply:
I just re-ran the setup and script with the patch and it did put the certs in there. Not sure why I was missing them before, but probably some issue with order of operations while testing this out. Thanks. I will re-commit with this removed and signed.